.jenkins/pipelines/Agnostic/Linux/Jenkinsfile

```diff
@@ -102,10 +81,6 @@ try{
         "Check CI":                                { checkCI() },
         "Check Developer Experience Ubuntu 16.04": { checkDevFlows('16.04') },
         "Check Developer Experience Ubuntu 18.04": { checkDevFlows('18.04') },
-        "AArch64 1604 GNU gcc Debug":              { AArch64GNUTest('16.04', 'Debug') },
-        "AArch64 1604 GNU gcc Release":            { AArch64GNUTest('16.04', 'Release') },
-        "AArch64 1804 GNU gcc Debug":              { AArch64GNUTest('18.04', 'Debug') },
```

Comment:
We should retain the aarch64 configurations. 1604 ones can be dropped later along with deprecating 1604 support.

Reply:
Should we maybe rename the 'gcc' part to not mislead?

I took a look at the code [here](https://github.com/openenclave/openenclave-ci/blob/master/src/jenkins/common/Openenclave.groovy#L66) which the tests rely on and it seems it doesn't have anything to do with gcc?

Reply:
It looks like
https://github.com/openenclave/openenclave/blob/3b6f413686846ec3b47ef7e94824c4078c2546f6/.jenkins/pipelines/Agnostic/Linux/Jenkinsfile#L51
will default to aarch gcc tool chain.

Reply:
I think your understanding of this matter may far exceed ours. If we are to deprecate gcc, that to me sounds like then this needs to be removed. Should the process instead be to deprecate gcc from docs and ci but keep these tests.

I looked into it a little, but it seems to me that GNU GCC != GCC which would align with what you are saying, whereas GNU GCC is for cross compilaiton.

Reply:
As I understand it, the reasoning is to keep GCC support for OPTEE (AArch64) even though we are deprecating it for SGX. I have pushed new changes that keeps this.